rest-server/bin/harness-env

```diff
@@ -3,14 +3,16 @@
 # Harness Server config, should work as-id unless you are using SSL
 # to listen on the host IP address for external connections
 # export REST_SERVER_HOST=0.0.0.0 # to listen for external connections
-export REST_SERVER_HOST=${REST_SERVER_HOST:-localhost}
+export REST_SERVER_HOST=${REST_SERVER_HOST:-localhost} # BEING DEPRECATED, USE HARNESS_URI
 # to change the port used
-export REST_SERVER_PORT=${REST_SERVER_PORT:-9090}
+export REST_SERVER_PORT=${REST_SERVER_PORT:-9090} # BEING DEPRECATED, USE HARNESS_URI
+export REST_SERVER_PROTOCOL=${REST_SERVER_PROTOCOL:-http} # BEING DEPRECATED, USE HARNESS_URI
+export HARNESS_URI=${HARNESS_URI:-${REST_SERVER_PROTOCOL}://${REST_SERVER_HOST}:${REST_SERVER_PORT}} # NOT USED
```

Comment:
after setting HARNESS_URI we need to extract REST_SERVER_HOST from the URI that way users can set via REST_SERVER_HOST or HARNESS_URI and it will work.

Eventually we remove setting by REST_SERVER_HOST when our containers support the URIs

This should be done for allo services.